x-pack/plugins/apm/server/routes/services/get_service_transaction_groups.ts

```diff
@@ -29,16 +29,20 @@ import {
 } from '../../lib/helpers/transaction_error_rate';
 
 const txGroupsDroppedBucketName = '_other';
+export const MAX_NUMBER_OF_TX_GROUPS = 1_000;
+
+export interface TransactionGroups {
+  transactionType: string;
+  name: string;
+  latency: number | null;
+  throughput: number;
+  errorRate: number;
+  impact: number;
+  alertsCount: number | undefined;
```

Comment:
Can you make this required? 

```suggestion
  alertsCount: number;
```